docs/UserGuide.md

```diff
@@ -137,26 +137,14 @@ Format: `editPerson INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [g/GENDER] [d
 - Existing values will be updated to the input values.
 - At least one of the optional fields must be provided.
 - Date format accepted is: dd/mm/yyyy.
+- Gender format accepted are: `M`/`m`/`Male`/`male` for male, `F`/`f`/`Female`/`female` for female.
 
 Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be
+*  `editPerson 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be
    `91234567` and `johndoe@example.com` respectively.
-*  `edit 3 n/Charlotte g/F d/3/4/1998` Edits the 3rd person’s contact: edits name to be `Charlotte`,
-edits gender to be `Female` and edits date of birth to be `3.4.1998`.
-
-__Optional Parameter 1: Gender__
-
-Format: `edit INDEX [g/GENDER]`
-
-- Edits the gender of a person in the contact list. The genders accepted by the contact list are: `M`/`m`/`Male`/`male`
-for male, `F`/`f`/`Female`/`female` for female.
-- `INDEX` must be **a positive integer** (i.e 1,2,3…)
-- `INDEX` must be within the range of the contact list index (i.e. from 1 to size of contact list).
-
-Examples:
-* `edit 1 g/M`
-* `edit 2 g/f`
-* `edit 3 g/F`
+*  `editPerson 2 n/Betsy Crower` Edits the name of the 2nd person to be `Betsy Crower`.
+*  `editPerson 3 n/Charlotte g/F d/3/4/1998` Edits the 3rd person’s name to be `Charlotte`,
```

Comment:
```suggestion
*  `editPerson 3 n/Charlotte g/F d/03/04/1998` Edits the 3rd person’s name to be `Charlotte`,
```
Has to be changed because the parser only takes in 2 digits for date and month now